plugins/pulp_python/plugins/models.py

```diff
@@ -1,120 +1,85 @@
-from gettext import gettext as _
 import hashlib
-import re
-import tarfile
 
 from mongoengine import StringField
 from pulp.server.db.model import FileContentUnit
+import twine  # noqa
```

Comment:
@jeremycline am I crazy? I had to add noqa here because flake8 said this was the incorrect order...